beekeeper-scheduler/src/main/java/com/expediagroup/beekeeper/scheduler/hive/HiveClient.java

```diff
@@ -75,6 +75,28 @@ public Map<String, PartitionInfo> getTablePartitionsInfo(String databaseName, St
     }
   }
 
+  public PartitionInfo getSinglePartitionInfo(String databaseName, String tableName, String partitionName) {
```

Comment:
what's partitionName?

Reply:
Add a line of javadoc please what the format is, so anyone calling knows what to expect. With an example.